lib/server.js

```diff
@@ -140,6 +141,7 @@ const createServer = ({
   const modules = {}
 
   const writeBundle = (baseUrl, bundle, { sourcemap } = {}) => {
+    console.time('server.writeBundle')
```

Comment:
Do we need this? Seems very verbose (log wise) to me.

Reply:
Forgot to remove that.